### PR TITLE
get_item and set_item impl

### PR DESCRIPTION
**Pull request by bartmachielsen**
See #360 for details about request

### CODE REVIEW

**Comment by Mogost** [comment]
I consider this change acceptable. But in order for this PR to be merged, it is necessary to document this feature.

File: constance/base.py [hunk 1]
```diff
@@ -9,6 +9,12 @@ def __init__(self):
         super(Config, self).__setattr__('_backend',
             utils.import_module_attr(settings.BACKEND)())
 
+    def __getitem__(self, key):
+        return self.__getattr__(key)
+    
+    def __setitem__(self, key, value):
+        return self.__setattr__(key, value)
+
```
Comment (Mogost) [inline constance/base.py]: In my opinion, this variation looks simpler.
```python
    __getitem__ = __getattr__
    __setitem__ = __setattr__
```

Comment (Mogost) [reply constance/base.py]: @bartmachielsen This must be declared after the associated methods are defined. Like that
```python
    def __getattr__(...):
        ...

    __getitem__ = __getattr__

    def __setattr__(...):
        ...

    __setitem__ = __setattr__
```

File: constance/base.py [hunk 2]
```diff
@@ -9,6 +9,9 @@ def __init__(self):
         super(Config, self).__setattr__('_backend',
             utils.import_module_attr(settings.BACKEND)())
 
+    __getitem__ = __getattr__
```
Comment (tayyab-razzaq) [inline constance/base.py]: it should be done after the `__getattr__` method definition

**Comment by jezdez** [comment]
Following closing the issue in https://github.com/jazzband/django-constance/issues/360#issuecomment-573616410 this PR isn't needed anymore.

**Pull request closed without merging**